From 973347dbc09bad42bf2ba957c96e70987b7b046c Mon Sep 17 00:00:00 2001
From: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
Date: Tue, 5 Mar 2024 10:12:52 -0300
Subject: [PATCH] csm-application-mobility: remove default values for new
 velero

This patch consists of removing the default values that were default
for old velero version, in new velero 6.0.0 these values need be
removed for helm

Signed-off-by: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
---
 charts/csm-application-mobility/values.yaml | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/charts/csm-application-mobility/values.yaml b/charts/csm-application-mobility/values.yaml
index a07282e..a9f59f4 100644
--- a/charts/csm-application-mobility/values.yaml
+++ b/charts/csm-application-mobility/values.yaml
@@ -32,11 +32,7 @@ objectstore:
 velero:
   enabled: true
   use-volume-snapshots: false
-  deployRestic: true
   cleanUpCRDs: true
-  restic:
-    # Set to true if installing on OpenShift
-    privileged: false
 
   credentials:
 #    Optionally, specify the name of the pre-created secret in the release namespace that holds the object store credentials. Either this or secretContents should be specified.
@@ -53,7 +49,6 @@ velero:
 # Based on the objectstore being used , the velero plugin and its configuration may need to change! Below is the sample configuration for using aws object store.
 # GCP and Azure plugins configuration are different. See more details at: https://github.com/vmware-tanzu/helm-charts/blob/main/charts/velero/README.md
   configuration:
-    provider: aws
     backupStorageLocation:
       name: default
       bucket: velero-bucket
-- 
2.34.1
